src/libs/ajax/workflows-app/Cbas.test.js

```diff
@@ -0,0 +1,337 @@
+import 'setimmediate';
```

Comment:
Has anything about our specific tests changed from CBAS UI to Terra UI changed, or am I safe to assume these are the same tests?

Reply:
safe assumption - should be no functional changes to the test